Tasks/DeployAzureResourceGroup/AzureResourceManagerHelper.ps1

```diff
@@ -134,11 +134,11 @@ function Get-Resources
                 }
         
                 #Adding WinRMHttp port property
-                if([string]::IsNullOrEmpty($winRmHttpPortMap[$resource.Name]) -eq $false)
-                {
-                    $property = New-Object Microsoft.VisualStudio.Services.DevTestLabs.Model.PropertyBagData($false, $winRmHttpPortMap[$resource.Name])
-                    $propertyBag.Add("WinRM_Http", $property)
-                }
+                #if([string]::IsNullOrEmpty($winRmHttpPortMap[$resource.Name]) -eq $false)
```

Comment:
Can we make this change as - 
if we find both HTTPS and HTTP - add HTTPS only, else add whatever is present.